auto_dev/contracts/contract.py

```diff
@@ -104,7 +104,7 @@ def update_contract_py(self) -> None:
         )
         contract_py = contract_py.replace(
             "from aea.configurations.base import PublicId",
-            f"from packages.{self.author}.contracts.{self.name} import PUBLIC_ID",
```

Comment:
we need this packages path in as the sys.path is appended to with the loaded components.